lib/SelectWallet.js

```diff
@@ -114,11 +131,54 @@ class SelectWallet extends Component {
     const accountTableData = this.buildSingleRowTable(accounts, 'Accounts');
 
     // evaluate safely in case undefined values exist
-    const _walletBalance = safeEval(() => new Amount(info[pluginState.selectedWallet].balance.confirmed).toBTC(), '');
-    const _accountBalance = safeEval(() => new Amount(balance[pluginState.selectedWallet][pluginState.selectedAccount].confirmed).toBTC(), '');
-    const _receiveAddress = safeEval(() => accountInfo[selectedWallet][selectedAccount].receiveAddress, '');
-    const _qrText = safeEval(() => accountInfo[selectedWallet][selectedAccount].receiveAddress, '');
+    const _walletBalance = safeEval(
+      () =>
+        new Amount(info[pluginState.selectedWallet].balance.confirmed).toBTC(),
+      ''
+    );
+    const _accountBalance = safeEval(
+      () =>
+        new Amount(
+          balance[pluginState.selectedWallet][
+            pluginState.selectedAccount
+          ].confirmed
+        ).toBTC(),
+      ''
+    );
+    const _receiveAddress = safeEval(
+      () => accountInfo[selectedWallet][selectedAccount].receiveAddress,
+      ''
+    );
+    const _qrText = safeEval(
+      () => accountInfo[selectedWallet][selectedAccount].receiveAddress,
+      ''
+    );
+    let expandedData = [{ mainData: [], subData: [] }];
+
+    if (txnHistory.length) {
+    }
 
+    // Setup TX History table
+    let txTable = <Text type="p">Must select a wallet and account to display transaction history</Text>;
```

Comment:
Move component creation to constructor to separate business logic and view logic